_posts/2021-05-28-state-of-the-union.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "State of the Union"
+date:   2021-05-28 16:45:00 +0100
+categories: introduction
+author:
+- Alexandra Settle, Volker Theile, and Alex Lau
+---
+
+Where are we at?
+
+Aquarium has made some good progress in the past few weeks. Here are the
+following recent highlights:
+
+- We now have a [feature-based development roadmap](https://github.com/aquarist-labs/aquarium/blob/main/doc/project-plan/roadmap.rst)
+- We refactored the installer: Now it uses the wizard style in both
+  scenarios (setting up new node and adding one to an existing cluster)
+- We have 48 GitHub stars and over 67,000 lines of code
+- Our website has had a facelift and we now store our meeting minutes
+  openly.
+
+Overall, we're in a really good place for Aquarium. Glass is based on Google's
+Angular & Angular Material framework. The entire code is written in TypeScript
+which is transpired into JavaScript to be able to run on various browsers.
```

Comment:
I suspect that, where it reads `transpired`, it should read `transpiled`.

Reply:
Good catch, cheers.